index.html

```diff
@@ -50,7 +53,179 @@ <h2>Software Developer</h2>
         need coded, don’t hestiate to contact me.
       </p>
     </section>
+    <!--headline sections ends here-->
 
-    <hr id="horizontal-line" />
+    <hr class="horizontal-line" />
+
+    <!--about sections-->
+    <section class="about-section">
+      <div>
+        <h3 class="title">Languages</h3>
+        <ul class="skills">
+          <li>Javascript</li>
+          <li>Python</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Frameworks</h3>
+        <ul class="skills">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Skills</h3>
+        <ul class="skills">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <!--about sections ends here-->
+
+    <hr class="horizontal-line" />
+
+    <!--section for social links and icons-->
+    <!--webhint flags section tags without heading as error.
+    Hence div tag is used to wrap the icons to pass the webhint check.
+    This approach is used since there is no heading given in the figma design on figma-->
+    <div class="social-icons">
+      <a href="#"><i class="fab fa-github"></i></a>
+      <a href="#"><i class="fab fa-linkedin-in"></i></a>
+      <a href="#"><i class="fab fa-instagram"></i></a>
+      <a href="#"><i class="fab fa-twitter"></i></a>
+      <a href="#"><i class="fab fa-facebook"></i></a>
+    </div>
+    <!--section for social links and icons ends here-->
+
+    <!--work section-->
+    <section class="work-section">
+      <div class="work-section-title-container">
+        <i class="fa fa-arrow-down work-icon"></i>
+        <h3 class="work-section-title">Work</h3>
+      </div>
+
+      <h1 class="recent-work">MY RECENT WORK</h1>
```

Comment:
- [ ] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.

Reply:
successfully changed h1 to h2 tag on